services/QuillLMS/app/models/cache_key.rb

```diff
@@ -0,0 +1,35 @@
+# Used to define common cache_keys in the app and force a common pattern
+module CacheKey
+
+  def self.all_classrooms(user, name:, groups: [])
+    classroom = user.classrooms_i_teach.maximum('classrooms.updated_at')
+
+    model_key(classroom, name: name, groups: groups)
+  end
+
+  def self.classroom(classroom, name:, groups: [])
```

Comment:
We can call `CacheKey.model_key` for these cases, since they are pure pass-through methods

Reply:
> We can call `CacheKey.model_key` for these cases, since they are pure pass-through methods

Yeah, see my longer comment about this. I'm aware this is a pass-through. My intention here was to mark `classroom` as a validated cache key that has all of the other updates in place to make it work properly. Happy to get feedback on that.